src/openrct2/world/footpath.c

```diff
@@ -385,6 +385,19 @@ static money32 footpath_place_real(sint32 type, sint32 x, sint32 y, sint32 z, si
         coord.y = y + 16;
         coord.z = map_element_height(coord.x, coord.y);
         network_set_player_last_action_coord(network_get_player_index(game_command_playerid), coord);
+
+        bool useDirection = !!(direction >> 8);
```

Comment:
Looks like you're trying to access the bit that you set when calling this command? In that case please explicitly mask it with the new mask define and then just shift.